Code/GraphMol/FileParsers/FileParserUtils.h

```diff
@@ -18,27 +18,50 @@
 #include <boost/algorithm/string.hpp>
 #include <boost/format.hpp>
 #include <RDGeneral/BoostEndInclude.h>
+#include <string_view>
 
 namespace RDKit {
 class RWMol;
 class Conformer;
 
 namespace FileParserUtils {
 template <typename T>
-T stripSpacesAndCast(const std::string &input, bool acceptSpaces = false) {
-  std::string trimmed = boost::trim_copy(input);
+T stripSpacesAndCast(std::string_view input, bool acceptSpaces = false) {
+  auto trimmed = input;
+  unsigned idx = 0u;
+  while (input[idx] == ' ') {
+    ++idx;
+  }
+  trimmed.remove_prefix(idx);
+  if (!trimmed.empty()) {
+    idx = trimmed.size() - 1;
+    while (trimmed[idx] == ' ') {
+      --idx;
+    }
+    trimmed.remove_suffix(trimmed.size() - idx - 1);
+  }
```

Comment:
Wouldn't this be easier with find_first/last not of?  Something like:

```
auto start = trimmed.find_first_not_of(' ');                                                                                                                               
  if (start != std::string_view::npos)                                                                                                                                       
    {                                                                                                                                                                        
      auto end = trimmed.find_last_not_of(' ') + 1;                                                                                                                          
      trimmed = trimmed.substr(start, end);                                                                                                                                  
    } else {                                                                                                                                                                 
    trimmed = "";                                                                                                                                                               
  }   
```

Reply:
You could also use the strip function below by passing in the whitespace to control behavior

```
strip(input, ' ')
strip(input, ' \r\t\n')
```

Reply:
Yep, that is simpler. Added it.